indico/modules/categories/models/categories.py

```diff
@@ -175,6 +175,11 @@ def __table_args__(cls):
         nullable=False,
         default=False
     )
+    show_future_months = db.Column(
+        db.Integer,
+        nullable=False,
+        default=1
```

Comment:
```suggestion
        default=0
```

matching the current behavior and the default set in the alembic revision

Reply:
Ah yes, I was going back and forth on this and missed changing it back. Thanks!